src/main/scala/com/databricks/labs/overwatch/pipeline/Schema.scala

```diff
@@ -725,6 +734,31 @@ object Schema extends SparkSessionWrapper {
     StructField("workspace_name", StringType, nullable = true)
   ))
 
+  val warehouseSnapMinimumSchema: StructType = StructType(Seq(
+    StructField("id", StringType, nullable = true),
```

Comment:
this SnapMinSchema is the schema AT REST in the target. In the snap land in bronze you renamed `id` to `warehouse_id` so this should be warehouse_id and it should not be nullable since it's a key

Reply:
done